Tests/SonarQube.TeamBuild.PostProcessor.Tests/MSBuildPostProcessorTests.cs

```diff
@@ -344,6 +346,30 @@ public void PostProc_WhenSettingInFileAndCommandLineArg_DoesNotFail()
             context.Logger.AssertErrorDoesNotExist(CredentialsErrorMessage);
         }
 
+        [TestMethod]
+        public void Execute_NullArgs_Throws()
+        {
+            AssertException.ExpectsNullArgumentException(
```

Comment:
We are using FluentAssertions everywhere so this bit of code is wrong IMO but let's say it's ok so you don't have to rewrite.